packages/web-components/fast-components/src/accordion-item/accordion-item.styles.ts

```diff
@@ -35,127 +34,112 @@ export const accordionItemStyles: (
     definition: AccordionItemOptions
 ) =>
     css`
-    ${display("flex")} :host {
-        box-sizing: border-box;
-        font-family: ${bodyFont};
-        flex-direction: column;
-        font-size: ${typeRampMinus1FontSize};
-        line-height: ${typeRampMinus1LineHeight};
-        border-bottom: calc(${strokeWidth} * 1px) solid ${neutralStrokeDividerRest};
-    }
-
-    .region {
-        display: none;
-        padding: calc((6 + (${designUnit} * 2 * ${density})) * 1px);
-    }
-
-    .heading {
-        display: grid;
-        position: relative;
-        grid-template-columns: auto 1fr auto calc(${heightNumber} * 1px);
-        z-index: 2;
-    }
-
-    .button {
-        appearance: none;
-        border: none;
-        background: none;
-        grid-column: 2;
-        grid-row: 1;
-        outline: none;
-        padding: 0 calc((6 + (${designUnit} * 2 * ${density})) * 1px);
-        text-align: left;
-        height: calc(${heightNumber} * 1px);
-        color: ${neutralForegroundRest};
-        cursor: pointer;
-        font-family: inherit;
-    }
-
-    .button:hover {
-        color: ${neutralForegroundRest};
-    }
-
-    .button:active {
-        color: ${neutralForegroundRest};
-    }
-
-    .button::before {
-        content: "";
-        position: absolute;
-        top: 0;
-        left: 0;
-        right: 0;
-        bottom: 0;
-        z-index: 1;
-        cursor: pointer;
-    }
-
-    .button:${focusVisible}::before {
-        outline: none;
-        border: calc(${focusStrokeWidth} * 1px) solid ${focusStrokeOuter};
-        border-radius: calc(${controlCornerRadius} * 1px);
-    }
-
-    :host([expanded]) .region {
+        ${display("flex")} :host {
+            box-sizing: border-box;
+            font-family: ${bodyFont};
+            flex-direction: column;
+            font-size: ${typeRampBaseFontSize};
+            line-height: ${typeRampBaseLineHeight};
+            color: ${neutralForegroundRest};
+            border-top: calc(${strokeWidth} * 1px) solid ${neutralFillActive};
+            border-bottom: calc(${strokeWidth} * 1px) solid ${neutralFillActive};
+            position: relative;
+        }
+        .region {
+            display: none;
+            padding: calc((6 + (${designUnit} * 2 * ${density})) * 1px);
+        }
+        .heading {
+            display: grid;
+            grid-template-columns: auto 1fr auto auto;
+            align-items: center;
+            z-index: 2;
+        }
+        .button {
+            appearance: none;
+            border: none;
+            background: none;
+            grid-column: 2;
+            grid-row: 1;
+            outline: none;
+            padding: 0 calc((6 + (${designUnit} * 2 * ${density})) * 1px);
+            text-align: left;
+            height: calc(${heightNumber} * 1px);
```

Comment:
Should line-height be included here with the same value as the height?

I don't think height is needed at all. Button is just a superficial wrapper around the contents of the heading.

Reply:
removing the height